config/helm/appmesh-controller/README.md

```diff
@@ -389,6 +389,8 @@ Parameter | Description | Default
 `tracing.address` |  Jaeger or Datadog agent server address (ignored for X-Ray) | `appmesh-jaeger.appmesh-system`
 `tracing.port` |  Jaeger or Datadog agent port (ignored for X-Ray) | `9411`
 `tracing.samplingRate` | X-Ray tracer sampling rate. Value can be a decimal number between 0 and 1.00 (100%)  | `0.05`
+`tracing.logLevel` | X-Ray agent log level, from most verbose to least: dev, debug, info, prod, warn, error. | `None`
```

Comment:
Shouldn't the default be one of the values ? what does None correspond to ?

Reply:
by default the log level is `"prod"` in XRay agent.
In `config/helm/appmesh-controller/values.yaml` & `config/helm/appmesh-controller/test.yaml` we are setting empty string `logLevel: ""` to this value. The reason for this I have mentioned in https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/554#discussion_r783350795

What's the best to set here?
1. keep this empty `""`
2. say `None`
3. or just put `prod`

Reply:
So as discussed, I am setting `prod` here and ignore if value is `prod` in the switch case.